### PR TITLE
Add changelog for 1.21.9

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,36 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix codeowner detected diff base branch to mergebase (#29783) (#29807)
+  * Fix Safari spinner rendering (#29801) (#29802)
+  * Fix missing translation on milestons (#29785) (#29789)
+  * Fix user router possbile panic (#29751) (#29786)
+  * Fix possible NPE in ToPullReviewList (#29759) (#29775)
+  * Fix the wrong default value of ENABLE_OPENID_SIGNIN on docs (#29925) (#29927)
+  * Solving the issue of UI disruption when the review is deleted without refreshing (#29951) (#29968)
+  * Fix loadOneBranch panic (#29938) (#29939)
+  * Fix invalid link of the commit status when ref is tag (#29752) (#29908)
+  * Only do counting when count_only=true for repo dashboard (#29884) (#29905)
```
Comment (wxiaoguang) [inline CHANGELOG.md]: This is not a bug fix.

**Review by silverwind — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lunny) [reply CHANGELOG.md]: 11e4c8d

**Comment by lunny** [comment]
wait #29972

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,39 @@ This changelog goes through all the changes that have been made in each release
...
+  * Solving the issue of UI disruption when the review is deleted without refreshing (#29951) (#29968)
+  * Fix loadOneBranch panic (#29938) (#29939)
+  * Fix invalid link of the commit status when ref is tagged (#29752) (#29908)
+  * Editor error message misleading due to re-used key. (#29859) (#29876)
+  * Fix double border and border-radius on empty action steps (#29845) (#29850)
+  * Use `Temporal.PlainDate` for absolute dates (#29804) (#29808)
+  * Fix incorrect package link method calls in templates (#29580) (#29764)
+  * Fix the bug that the user may log out if GetUserByID returns unknown error (#29962) #29964
+  * Performance improvements for pull request list page (#29900) #29972
+  * Fix bugs in rerunning jobs (#29983) #29955
```
Comment (wolfogre) [inline CHANGELOG.md]:
```suggestion
  * Fix the bug that the user may log out if GetUserByID returns unknown error (#29962) (#29964)
  * Performance improvements for pull request list page (#29900) (#29972)
  * Fix bugs in rerunning jobs (#29983) (#29955)
```

**Review by wolfogre — APPROVED** [review]

**Pull request merged**